core-models/src/main/scala/com/mesosphere/usi/core/models/ReservationRecord.scala

```diff
@@ -0,0 +1,13 @@
+package com.mesosphere.usi.core.models
+
+/**
+  * * ReservationRecord is an immutable event of created reservations .
+  * @param sequenceNr number used for ordering the reservation record events.
+  * @param reservationId id of the reservation
+  * @param agentId id of the agent
+  * @param resources reserved resources
+  */
+case class ReservationRecord(sequenceNr: Long,
```

Comment:
Same as for `unserviceable` – I'd leave this whole class out for now since we're not going to use/implement it right away. It'll just create clutter that needs to be touched and refactored. Let's add it once we add support for persistent volumes..